code/modules/language/piratespeak.dm

```diff
@@ -10,3 +10,4 @@
 	"shiver","timbers","matey","swashbuckler"
 	)
 	icon_state = "pirate"
+	secret = TRUE // SKYRAT EDIT
```

Comment:
Just add that in a modular file, by just doing this:
```
/datum/language/piratespeak
    secret = true```

I don't remember by heart what kind of files we already have for languages, but you should modularize this, there's no reason it's not modular.